### ISSUE TITLE
Issue syncing age restricted videos

### ISSUE THREAD

**Issue by zeeshanakram3** [Joystream/youtube-synch#190]
## Context

YT-synch system syncs all the `public` videos of a Youtube channel. Even though the video(https://www.youtube.com/watch?v=9IX3czniqL8) is `public`, it has an age restriction on it. So downloading it throws the following error:

```
"message": "ERROR: [youtube] 9IX3czniqL8: Sign in to confirm your age. This video may be inappropriate for some users.",
```

## Fix

Either find a way to bypass the age restriction filter while downloading the video or just exclude age-restricted videos from syncing.

**Comment by bedeho** [comment]
Nice find, let us def. just exclude for now. I am not even sure what counts as age restricted on YouTube, but not a prio for us or any other app right now to my knowledge.

**Comment by zeeshanakram3** [comment]
addressed in #193
